port docs from engine wiki

stakach opened the pull request.

Commit:
port docs from engine wiki

stakach commented:
@KimBurgess I'm not sure how to fix the failure...

KimBurgess commented:
Oh yeah, I see what you mean - that output is useless.
It looks like the proofer isn't providing the actual analytics / assist text. Let me take a look why.

kimburgess committed:
Merge branch 'master' into driver-development

KimBurgess commented:
@stakach proof reader should be a little more useful now. To resolve the spelling errors ensure field names are enclosed in code blocks (or backticks) when referenced. Technical terms which should be included in the grammar can be pushed into `.spelling`.

kimburgess committed:
Merge branch 'master' into driver-development

Commit:
(package) update NPM deps

Commit:
(disallowed-phrases) remove extron

we are pretty friendly with them as they have open protocols and they
are mentioned neutrally at the moment.

Commit:
(spelling) add all the words

Commit:
(docs) fix major spelling and errors flagged by build tools

Commit:
Merge remote-tracking branch 'origin/driver-development' into driver-development

# Conflicts:
#	src/analysers/disallowed-phrases.ts
